src/element-templates/properties-panel/properties/CustomProperties.js

```diff
@@ -919,6 +919,10 @@ function propertyValidator(translate, property) {
       return translate('Must not be empty.');
     }
 
+    if (isFeel(value)) {
+      return;
+    }
+
```

Comment:
We do have access to the property, so we could theoretically add stricter validation with

```suggestion
    if (property.feel && isFeel(value)) {
      return;
    }
```

With this, this would still apply patterns to properties that are normal string type without feel expressions.

Reply:
According to the [schema](https://github.com/camunda/element-templates-json-schema/blob/main/packages/zeebe-element-templates-json-schema/src/defs/properties.json#L459) the `feel` property can also be `static`. Would that mean the property is _not_ FEEL? In that case we'd need to check the value of the property.

Reply:
Answered. `static` still means it's a FEEL expression. So a simple check is enough.